worker/pkg/zip/zip_test.go

```diff
@@ -0,0 +1,68 @@
+package zip
+
+import (
+	"archive/zip"
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Buffer implements io.ReadAtCloser and io.WriteCloser
+type Buffer struct {
+	bytes.Buffer
+}
+
+func (*Buffer) Close() error {
+	return nil
+}
+
+func TestNewUnzipper(t *testing.T) {
+	zf, err := os.Open("testdata/test.zip")
+	require.NoError(t, err)
+	fInfo, err := zf.Stat()
+	require.NoError(t, err)
+
+	wFn := func(name string) (io.WriteCloser, error) {
+		return &Buffer{bytes.Buffer{}}, nil
+	}
+	r, err := zip.NewReader(zf, fInfo.Size())
+	assert.NoError(t, err)
+	_, err = NewUnzipper(r, wFn)
+	assert.NoError(t, err)
+}
+
+func TestUnzipper_Unzip(t *testing.T) {
+	zf, err := os.Open("testdata/test.zip")
+	require.NoError(t, err)
+
+	fInfo, err := zf.Stat()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	expectedFiles := map[string]*Buffer{"test1.txt": {*bytes.NewBufferString("hello1")}, "test2.txt": {*bytes.NewBufferString("hello2")}}
+
+	//map of buffers which will keep unzipped data
+	files := map[string]*Buffer{"test1.txt": {bytes.Buffer{}}, "test2.txt": {bytes.Buffer{}}}
+	wFn := func(name string) (io.WriteCloser, error) {
+		return files[name], nil
+	}
+
+	r, err := zip.NewReader(zf, fInfo.Size())
+	require.NoError(t, err)
+	uz, err := NewUnzipper(r, wFn)
+	require.NoError(t, err)
+
+	log.Default().Print(expectedFiles, uz)
+	err = uz.Unzip()
+	assert.NoError(t, err)
+	for k, v := range files {
+		assert.Equal(t, v.Bytes(), expectedFiles[k].Bytes())
+	}
+
+}
```

Comment:
It is also better to test if the Unzip method itself returns the same error when writerFn returns an error. 

Also, to test if the zip bomb attack countermeasure is effective, we could combine a Reader that spits out the same value forever with a Writer that discards the value (optional, as it increases the difficulty a bit).

Reply:
fixed wtih [add test to check error](https://github.com/reearth/reearth-cms/pull/205/commits/e2a2d668e41f76afc910c22cff6286eb0892eba3)

Reply:
Since I reversed expected value and actual value, I've fixed it with
[reverse expected and actual](https://github.com/reearth/reearth-cms/pull/205/commits/71ec0544f2445f9809928650b9000e3f33f50b1f)